src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -36,7 +36,7 @@
     private static final String REGEX_BLOB_KEY = "(encoded_gs_key:)?[a-zA-Z0-9-_]{10,}";
     private static final String REGEX_QUESTION_ID = "[a-zA-Z0-9-_]{40,}";
     private static final String REGEX_COMMENT_ID = "[0-9]{16}";
-    private static final String REGEX_DISPLAY_TIME = "(0[0-9]|1[0-2]):[0-5][0-9] [AP]M( UTC)?";
+    private static final String REGEX_DISPLAY_TIME = "(0[0-9]|1[0-2]):[0-5][0-9] [AP]M( -+[0-9])?";
```

Comment:
The code will need to do more than just a regex replace. It'll have to parse the timestamp, convert it to UTC, and check whether it's the same as the current time. If it is, replace with `{datetime.now+X}` where X is the correct time zone.